packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenters/datacenters.html

```diff
@@ -75,15 +75,30 @@ <h2 data-translate="dedicatedCloud_datacenters_title"></h2>
             >
             </span>
         </oui-datagrid-column>
+
         <oui-datagrid-column
-            data-title=":: 'dedicatedCloud_datacenters_cluster' | translate"
+            data-title=":: 'dedicatedCloud_datacenters_number_vm' | translate"
         >
-            <span data-ng-bind="$row.clusterSize"></span>
+            <span data-ng-if="!$row.error" data-ng-bind="$row.nbreVm"></span>
+            <span
+                class="text-danger"
+                data-ng-if="$row.error"
+                data-ng-bind="'dedicatedCloud_datacenters_error' | translate"
+            ></span>
         </oui-datagrid-column>
+
         <oui-datagrid-column
-            data-title=":: 'dedicatedCloud_datacenters_edges' | translate"
+            data-title=":: 'dedicatedCloud_datacenters_number_vm_licensed' | translate"
         >
-            <span data-ng-bind="$row.edgesCount"></span>
+            <span
+                data-ng-if="!$row.error"
+                data-ng-bind="$row.nbreVmLicensed"
```

Comment:
```suggestion
                data-ng-bind="$row.nbVmLicensed"
```

nbre is too near french typo